src/pages/HomePage.tsx

```diff
@@ -182,51 +187,55 @@ function HomePage({event}: IProps & RouteComponentProps) {
                 content={`twitch.tv/${event.meta.twitchChannel}`}
                 link
               />
-              {/*<HeaderMetaList>*/}
-              {/*  <HeaderLinks href="https://esamarathon.com/news/e7a9a8a5-658a-4eea-a2f9-5b178a812be4">*/}
-              {/*    Master Post*/}
-              {/*  </HeaderLinks>*/}
-              {/*  <HeaderLinks href="https://esamarathon.com/rules">Code of Conduct</HeaderLinks>*/}
-              {/*  <HeaderLinks href="https://esamarathon.com/news/5ec16dac-492c-4fa3-9ac4-1bcf896aadbb">*/}
-              {/*    Attendee Guide*/}
-              {/*  </HeaderLinks>*/}
-              {/*</HeaderMetaList>*/}
+              <HeaderMetaList>
```

Comment:
We shouldn't show these atm, since they are linking to old outdated stuff.

Reply:
Ah yes, I temporarily added it cuz it somehow broke the home page header slidey thingy.